client/lib/analytics/index.js

```diff
@@ -61,15 +63,16 @@ function buildQuerystringNoPrefix( group, name ) {
 // we use this variable to track URL paths submitted to analytics.pageView.record
 // so that analytics.pageLoading.record can re-use the urlPath parameter.
 // this helps avoid some nasty coupling, but it's not the cleanest code - sorry.
-var mostRecentUrlPath = null;
-
-window.addEventListener('popstate', function() {
-	// throw away our URL value if the user used the back/forward buttons
-	mostRecentUrlPath = null;
-});
+let mostRecentUrlPath = null;
 
-var analytics = {
+if ( typeof window !== 'undefined' ) {
```

Comment:
yup and for test mocking. it's a pain to run the tests in node and have to fake out window just to get the tests to load.